promoted_python_delivery_client/client/delivery_request.py

```diff
@@ -0,0 +1,13 @@
+from dataclasses import dataclass
+from typing import Optional
+from promoted_python_delivery_client.client.insertion_page_type import InsertionPageType
+from promoted_python_delivery_client.model.cohort_membership import CohortMembership
+from promoted_python_delivery_client.model.request import Request
+
+
+@dataclass
+class DeliveryRequest:
+    request: Request
+    experiment: Optional[CohortMembership] = None
+    only_log: Optional[bool] = False
```

Comment:
Why optional if you're providing a default? Does explicitly setting to None have a different effect than accepting the False default?

Reply:
Note that this is the case throughout this PR so feel free to update everywhere or ignore. I've removed the other comments since it's all the same sentiment.

Reply:
In the model package, they need to be Optional in order to know to omit them from JSON serialization as we want to save bandwidth.

In the client package for the non-object Optionals with defaults, I'll change them (e.g. line 12, there were only a couple). For the object (e.g. line 11) I feel like this is preferable to a union annotation of CohortMembership | None. But maybe I'm wrong on this style?